mentorship ios/ViewModels/DetailListCellViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DetailListCellViewModel.swift
+//  Created on 22/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+import Combine
+
+class DetailListCellViewModel: ObservableObject {
+    var requestData: RequestStructure
```

Comment:
do we need all information from RequestStructure to display data?

Reply:
I've now tailored all the view models (till relation view model) and have pushed the commit.